aspnet/signalr/overview/performance/scaleout-with-windows-azure-service-bus.md

```diff
@@ -103,51 +103,51 @@ In your application's Startup.cs file, add the following code:
 
 Now you need to get your service bus connection string. In the Azure portal, select the service bus namespace that you created and click the Access Key icon.
 
-![An image of a plus sign labeled 'Create', an image of a key labeled 'Access Key', and an image of a trash can labeled 'Delete'.](scaleout-with-windows-azure-service-bus/_static/image7.png)
+![Graphic of a plus sign labeled Create and a key labeled Access Key and a trash can labeled Delete.](scaleout-with-windows-azure-service-bus/_static/image7.png)
 
 Copy the connection string to the clipboard, then paste it into the *connectionString* variable.
 
-![Window titled 'Connect to your namespace' with options underneath.](scaleout-with-windows-azure-service-bus/_static/image8.png)
+![Screenshot of Window titled Connect to your namespace with options underneath.](scaleout-with-windows-azure-service-bus/_static/image8.png)
 
 [!code-csharp[Main](scaleout-with-windows-azure-service-bus/samples/sample4.cs)]
 
 ## Deploy to Azure
 
 In Solution Explorer, expand the **Roles** folder inside the ChatService project.
 
-![The folder 'Roles' is open and 'SignalRChat' is selected.](scaleout-with-windows-azure-service-bus/_static/image9.png)
+![Screenshot of an open folder titled Roles and Signal R Chat selected.](scaleout-with-windows-azure-service-bus/_static/image9.png)
 
 Right-click the SignalRChat role and select **Properties**. Select the **Configuration** tab. Under **Instances** select 2. You can also set the VM size to **Extra Small**.
 
-![Under Instances, the Instance count is set to 2 and VM Size is set to Extra small.](scaleout-with-windows-azure-service-bus/_static/image10.png)
+![Screenshot showing Instances and the Instance count is set to 2 and V M Size is set to Extra small.](scaleout-with-windows-azure-service-bus/_static/image10.png)
 
 Save the changes.
 
 In Solution Explorer, right-click the ChatService project. Select **Publish**.
 
-![A menu with 'Publish' selected.](scaleout-with-windows-azure-service-bus/_static/image11.png)
+![Screenshot of Chat Service menu with Publish selected.](scaleout-with-windows-azure-service-bus/_static/image11.png)
```

Comment:
How about, "Screenshot that shows Solution Explorer. Publish is selected in the Chat Service context menu."